### PR TITLE
Cleanup test case

### PR DESCRIPTION
**Pull request by MorrisJobke**
Stream lines the tests with https://github.com/owncloud/core/pull/25346

cc @rullzer @nickvergessen @LukasReschke

### CODE REVIEW

**Comment by nickvergessen** [comment]
👍

**Review by ChristophWurst — APPROVED** [review]
👍

**Review by rullzer — APPROVED** [review]

File: tests/lib/Share20/DefaultShareProviderTest.php [hunk 1]
```diff
@@ -484,7 +485,8 @@ public function testDeleteGroupShareWithUserGroupShares() {
 		$share->method('getId')->willReturn($id);
 		$share->method('getShareType')->willReturn(\OCP\Share::SHARE_TYPE_GROUP);
 
-		$provider = $this->getMockBuilder('OC\Share20\DefaultShareProvider')
+		/** @var DefaultTokenMapper $provider */
```
Comment (rullzer) [inline tests/lib/Share20/DefaultShareProviderTest.php]: fixed

**Comment by rullzer** [comment]
LGTM

**Pull request merged**